libpd_wrapper/z_hooks.c

```diff
@@ -10,6 +11,7 @@
  
 #include "z_hooks.h"
 
+// libpd_printhook is sys_printhook
```

Comment:
This seems misleading. I think the point is that there is no libpd_printhook because sys_printhook already serves this purpose.

Reply:
Removed. This info is already in the header anyway.